cfgov/unprocessed/js/modules/YoutubePlayer.js

```diff
@@ -0,0 +1,84 @@
+/* ==========================================================================
+   YouTube Initialization
+   ========================================================================== */
+
+'use strict';
+
+var _noopFunct = require( './util/standard-type' ).noopFunct;
+var _noopFunct = require( './util/standard-type' ).noopFunct;
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/pull/1625/commits/36f6195becc4c28009ba7b84c31290ffe07277ae